include/spirv-tools/libspirv.h

```diff
@@ -353,6 +339,9 @@ typedef enum {
   SPV_ENV_VULKAN_1_0_7      // Vulkan 1.0 revision 7.
 } spv_target_env;
 
+// Returns a string describing the given SPIR-V target environment.
+const char* spvTargetEnv(spv_target_env env);
```

Comment:
I think I want the name of the function to indicate that it returns a name for the given enum.
Something like  spvTargetEnvDescription